src/views/colours.njk

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
Is it worth including in the docs which colours to use for specific contexts? E.g., links; success/error/focus states etc

Reply:
Yeah I haven't done that just because we haven't implemented anything yet but will definitely want to include that stuff soon